shell.nix

```
@@ -516,12 +528,13 @@ let
       cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)/
       ${pkgs.nodePackages.nodemon}/bin/nodemon --delay 200ms -e hs,yaml --watch server/src --watch server/package.yaml --watch clientmodel/lib/src --watch clientmodel/lib/package.yaml --exec run-server-inner
     '')
-	(pkgs.writeScriptBin "watch-remix" ''
+    (pkgs.writeScriptBin "watch-remix" ''
       #!/usr/bin/env bash
       set -e
       cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)/utopia-remix
       ${pnpm}/bin/pnpm install
-	  PORT=8002 pnpm run dev
+      ${pnpm}/bin/pnpm exec prisma generate
+      PORT=8002 pnpm run dev
```

```suggestion
      PORT=8002 ${pnpm}/bin/pnpm run dev
```
Double check this works before accepting the suggestion, but there's no reason why it shouldn't. The reason for this is that it ensures this step is using the exact same version of `pnpm` as all others, as opposed to another version that may happen to be on the PATH

---

yesss I forgot to update this one 🤦

---

https://github.com/concrete-utopia/utopia/pull/4846/commits/48b6821ce59a623c0a4dc7d6dcc9d2035f531cd7